### PR TITLE
qa/tasks/cephadm: enable mon_cluster_log_to_file

### PR DESCRIPTION
**Pull request by dvanders**
Without cluster_log_to_file we have nothing to grep for errors:

2023-10-27T16:06:59.111 DEBUG:teuthology.orchestra.run.smithi150:> sudo egrep '\[ERR\]|\[WRN\]|\[SEC\]' /var/log/ceph/38cc7fce-74d9-11ee-8db9-212e2dc638e7/ceph.log | egrep -v '\(MDS_ALL_DOWN\)' | egrep -v '\(MDS_UP_LESS_THAN_MAX\)' | head -n 1 2023-10-27T16:06:59.141 INFO:teuthology.orchestra.run.smithi150.stderr:grep: /var/log/ceph/38cc7fce-74d9-11ee-8db9-212e2dc638e7/ceph.log: No such file or directory

Set mon_cluster_log_to_file = true.

Fixes: https://tracker.ceph.com/issues/63425

See https://pulpito.ceph.com/teuthology-2023-10-28_14:23:03-upgrade:quincy-x-reef-distro-default-smithi/7439369/ for a broken example

### CODE REVIEW

File: qa/tasks/cephadm.conf
Comment (adk3798) [inline qa/tasks/cephadm.conf:6]: Does this serve the same purpose as https://github.com/ceph/ceph/pull/48539 ?

Comment (idryomov) [reply qa/tasks/cephadm.conf:6]: We still need https://github.com/ceph/ceph/pull/48539 in case something else goes sideways.

@adk3798 Please prioritize adding necessary allowlists to cephadm suite -- it's what that PR is blocked on and why I have been unstaling it for months now.  Chris and I didn't want to merge something that would turn one of the major suites red, but there a limit to how long something like that can wait...

Comment (idryomov) [reply qa/tasks/cephadm.conf:6]: > I couldn't confirm if this cephadm.conf change makes the ceph.log appear -- can you please help testing that?

@dvanders If it does, it would turn cephadm suite red because, unlike in other suites, there are no ignorelists there.

Comment (adk3798) [reply qa/tasks/cephadm.conf:6]: @idryomov  At this point, if it's causing failures to be missed elsewhere, I'm okay with it being merged. I have a partially complete ignorelist PR and this will at least force me to finish it. I can deal with sifting through the failures for the time before I finish that as well.

Comment (idryomov) [reply qa/tasks/cephadm.conf:6]: @adk3798 I have asked @chrisphoffman to move forward on https://github.com/ceph/ceph/pull/48539.

Comment (idryomov) [reply qa/tasks/cephadm.conf:6]: > @adk3798 I have asked @chrisphoffman to move forward with #48539.

This hasn't happened.  I have unstaled #48539 again, but would suggest moving forward by picking up this PR and seeing if it actually makes ceph.log file appear and causes those ignorelist-related failures.

Comment (idryomov) [reply qa/tasks/cephadm.conf:6]: > If it does, it would turn cephadm suite red

Adam is asking for another run, but it's definitely working:

https://pulpito.ceph.com/yuriw-2024-01-31_15:48:29-rados:cephadm-main-distro-default-smithi/

Let's get this merged!

**Review by idryomov — APPROVED** [review]

**Comment by idryomov** [comment]
jenkins test api

**Review by adk3798 — APPROVED** [review]
https://pulpito.ceph.com/yuriw-2024-01-31_19:13:29-orch:cephadm-main-distro-default-smithi/

quite red, as expected in this case.

**Comment by ljflores** [comment]
There are a lot of failures in the rados suite from this PR. Until a PR is raised to whitelist expected warnings, a lot of main test batches are blocked.

See https://tracker.ceph.com/issues/64343.

Should be a relatively straightfoward fix from the rados team, but pasting here to raise awareness.

**Comment by ljflores** [comment]
@dvanders the rados team decided it would be best to revert this commit to unblock testing efforts for Squid, with the intent of re-merging the commit with some additions to the allowlist.

https://github.com/ceph/ceph/pull/55498

**Pull request merged**
